SETUP.md

```diff
@@ -115,7 +115,7 @@ git rm .appveyor.yml
 git rm ci/install.sh
 ```
 
-GitHub Actions is _usually_ able to deploy without any setup using the [`GITHUB_TOKEN`](https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token) for authentication.
+GitHub Actions is _usually_ able to deploy without any additional setup using the [`GITHUB_TOKEN`](https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token) for authentication.
```

Comment:
@ctb one question on your GitHub Actions experience (unrelated to this PR). Did you have to do the following to get the GitHub Pages website to deploy?

> try reselecting "gh-pages branch" as the Source for GitHub Pages in the repository Settings. GitHub Pages should now trigger on the next commit.

also described [independently here](https://github.com/peaceiris/actions-gh-pages/tree/b4f6cbe8b15af25d3f1b5e3e11066669a792ccae#%EF%B8%8F-first-deployment-with-github_token).

It's possible GitHub has fixed this bug.

refs https://twitter.com/ctitusbrown/status/1311332637743538177